src/main/kotlin/io/pivotal/intellij/jasmine/util/JasmineFileUtil.kt

```diff
@@ -0,0 +1,38 @@
+package io.pivotal.intellij.jasmine.util
+
+import com.intellij.lang.javascript.psi.JSFile
+import com.intellij.openapi.vfs.VirtualFile
+
+/**
+ * Utility class for handling Jasmine files
+ */
+object JasmineFileUtil {
+    /**
+     * Checks if a JavaScript file is a Jasmine test file
+     * Uses file name patterns and content analysis
+     */
+    fun isJasmineTestFile(jsFile: JSFile): Boolean {
+        val virtualFile = jsFile.virtualFile ?: return false
+        
+        // First check file name patterns
+        if (hasJasmineTestFileName(virtualFile)) {
+            return true
+        }
+        
+        // Then check file content for Jasmine test constructs
+        val fileContent = jsFile.text.lowercase()
+        return fileContent.contains("describe(") && 
+               (fileContent.contains("it(") || fileContent.contains("test(")) &&
+               (fileContent.contains("expect(") || fileContent.contains("jasmine."))
+    }
+    
+    /**
+     * Checks if a file has a Jasmine test file name pattern
+     */
+    fun hasJasmineTestFileName(file: VirtualFile): Boolean {
+        val fileName = file.nameWithoutExtension.lowercase()
+        return fileName.endsWith(".spec") || 
+               fileName.endsWith(".test") || 
+               fileName.contains("spec")
```

Comment:
It doesn't make any sense to check for a fixed set of strings, since a Jasmine spec file is any file that matches the user's configured list of globs. But even if the approach made sense, I'm not sure why one of the three should be different from the rest.

It might be best to remove this function entirely and assume that any file that matches the remaining heuristics could be a Jasmine spec file